docs/overviews/overview-publishers.md

```diff
@@ -43,6 +43,7 @@ The following documentation resources are available for publishers to implement
 | :--- | :--- | :--- | :--- |
 | Client-Side (Web) SDK | [UID2 SDK for JavaScript](../sdks/client-side-identity.md) | Client-Side JavaScript SDK that facilitates the process of establishing client identity using UID2 and retrieving advertising tokens for publishers. | Publishers |
 | Client-Side (Web) Integration Guide | [UID2 SDK for JavaScript Integration Guide](../guides/publisher-client-side.md) | This integration guide for publishers covers standard web integration scenarios that use the [UID2 SDK for JavaScript](/docs/sdks/client-side-identity.md). | Publishers |
+| Prebid | [Prebid Integration Guide](../guides/integration-prebid.md) | An integration guide for publishers who want to directly integrate with UID2 and generate identity tokens to be passed by Prebid in the RTB bid stream. | Publishers |
```

Comment:
same here - move this line to top of the table?

Reply:
Done thx.